netbox/templates/dcim/device.html

```diff
@@ -300,6 +300,29 @@ <h5 class="card-header">Services</h5>
             </div>
             {% include 'inc/panels/contacts.html' %}
             {% include 'inc/panels/image_attachments.html' %}
+            <div class="card">
+                <h5 class="card-header">Dimensions</h5>
+                <div class="card-body table-responsive">
+                    <table class="table table-hover attr-table">
+                        <tr>
+                            <th scope="row">Height</th>
+                            <td>
+                                {{ object.device_type.u_height }}U ({% if object.desc_units %}descending{% else %}ascending{% endif %})
```

Comment:
I don't think the "descending" indicator makes sense here. (This was probably copied from the rack template by mistake.)

```suggestion
                                {{ object.device_type.u_height }}U
```